lint/build.gradle.kts

```diff
@@ -20,8 +20,8 @@ plugins {
 }
 
 java {
-    sourceCompatibility = JavaVersion.VERSION_1_8
-    targetCompatibility = JavaVersion.VERSION_1_8
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
```

Comment:
Could you give more information about the benefit of using Java 11 than Java 8? 🤔

Reply:
@trietbui85 can use more java API from 9 to 11 when enabling JDK desugaring

_Sent from my 2201117TG using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_